docs/building.rst

```diff
@@ -182,7 +189,7 @@ can set the ``runtime-link`` feature on the command line, either to ``shared``
 or ``static``. Most operating systems will only allow linking shared against
 the runtime, but on windows you can do both. Example::
 
-  b2 msvc-14.0 link=static runtime-link=static
+  b2 msvc-14.2 link=static runtime-link=static debug-symbols=on
```

Comment:
`debug-symbols=on` is on by default (since the default build variant is `debug`).

Reply:
ok, I added variant=release, because I wanted to give an example of how to build with debug symbols (because I think this is what most people want to do - you always want to keep debug symbols to analyze crashes in production)